rzk/grammar/Syntax.cf

```diff
@@ -78,6 +78,10 @@ ParamVarShapeDeprecated. ParamDecl ::= "{" "(" Pattern ":" Term ")" "|" Term "}"
 paramVarShapeDeprecated. ParamDecl ::= "{" Pattern ":" Term "|" Term "}" ;
 define paramVarShapeDeprecated pat cube tope = ParamVarShapeDeprecated pat cube tope ;
 
+-- Parameter declaration for Sigma types 
+SigmaParam. SigmaParam ::= Pattern ":" Term ;
```

Comment:
Ideally, I would like to reuse `Param`, but I think this should be a separate change, since `Param` allows shapes and we do not support shape types yet (see #50).